tour/authorizations-code.md

```diff
@@ -6,15 +6,15 @@ Below is a solution for the exercise.
 
 ```java
 static void exercise(AccumuloClient client) throws Exception {
-    // create a table called "GothamPD".
+    // Create a table called "GothamPD".
```

Comment:
:+1: These make things a little more consistent. Normally, this wouldn't matter, but since this is intended to be example code that users following along the tour will use, it can help make the tour more presentable.